image-customize

```diff
@@ -140,14 +141,18 @@ class BuildAction(ActionBase):
         if opt_quick:
             mock_opts += ' --nocheck'
 
-        # build source package
-        machine.execute(f'''su builder -c 'rpmbuild --define "_topdir /var/tmp/build" -ts "{vm_source}"' ''')
```

Comment:
Why not this:

```
   flag = 'r' if vm_source.endswith('.src.rpm') else 't'
   machine.execute(f'''... -{flag}s {vm_source} ...''')
```
?

Reply:
FTR: It may make the code smaller, but I find it harder to understand then -- we already have an srpm, and I'd like it to build the srpm exactly as given; I am not sure if rebuilding an srpm from an srpm gives you the exact same result, and also it just feels like a waste of time, and loss of predictability.